packages/react/src/components/List/List.tsx

```diff
@@ -403,6 +403,10 @@ export class List<T = any> extends React.Component<IListProps<T>, IListState<T>>
       return true;
     }
 
+    if (newProps.className !== this.props.className) {
```

Comment:
Would you please add a new test or a new assertion to this [className test](https://github.com/microsoft/fluentui/blob/master/packages/react/src/components/List/List.test.tsx#L174) to verify this fix?

Reply:
Done. I updated existing test, which actually was verifying the className being updated, but because it also changed the items, the list would rerender. Now this test fails without my fix.